code/game/gamemodes/clock_cult/clock_structures/ratvar_the_clockwork_justicar.dm

```diff
@@ -69,49 +69,48 @@
 	for(var/mob/living/L in GLOB.living_mob_list) //we want to know who's alive so we don't lose and retarget a single person
 		if(L.z == z && !is_servant_of_ratvar(L) && L.mind)
 			meals += L
+	if(GLOB.cult_narsie && GLOB.cult_narsie.z == z)
+		meals = list(GLOB.cult_narsie) //if you're in the way, handy for him, but ratvar only cares about nar-sie!
+		prey = GLOB.cult_narsie
+		if(get_dist(src, prey) <= 10)
+			clash()
+			return
 	if(!prey)
-		for(var/obj/singularity/narsie/N in GLOB.singularities)
-			if(N.z == z)
-				prey = N
-				break
 		if(!prey && LAZYLEN(meals))
+			var/mob/living/L = prey
 			prey = pick(meals)
 			to_chat(prey, "<span class='heavy_brass'><font size=5>\"You will do, heretic.\"</font></span>\n\
-			<span class='userdanger'>You feel something massive turn its crushing focus to you...</span>")
-			prey << 'sound/effects/ratvar_reveal.ogg'
+			<span class='userdanger'>You feel something massive turn its crushing hatred onto you...</span>")
```

Comment:
To be clear here I mean atomize as in I disagree greatly with these text changes and would rather not destroy the entire pr over it.